manual/C/ch_Finance-Quote.xml

```diff
@@ -117,15 +122,26 @@ Translators:
         </para>
       </step>
     </procedure>
+
+    <tip>
+      <para>Afterwards it&rsquo;s a good idea to include the folder where &app; is installed,
+        under &win; e.g. <userinput>c:\Programme<replaceable> (x86)</replaceable>gnucash\bin</userinput>,
```

Comment:
"Programme" is English English, American English is "Program". The actual path on en_US localized Windows is `C:\Program Files (x86)\gnucash\bin`.